Issues from try run:

### Windows: maybe-configure Error 1. Perhaps related to previous error: Python not found 

```
Python was not found; run without arguments to install from the Microsoft Store, or disable this shortcut from Settings > Manage App Execution Aliases.
  cat: config.log: No such file or directory
  mozmake: *** [C:\Users\servo\.cargo\git\checkouts\mozjs-76be79f630786c2e\439c9fe\mozjs-sys\makefile.cargo:161: maybe-configure] Error 1
  thread 'main' panicked at C:\Users\servo\.cargo\git\checkouts\mozjs-76be79f630786c2e\439c9fe\mozjs-sys\build.rs:345:5:
  assertion failed: result.success()
```
### Android x86: 
```
error[E0080]: evaluation of constant value failed
    --> /home/runner/work/servo/servo/target/i686-linux-android/release/build/mozjs_sys-efd0814d4e0d1b70/out/build/jsapi.rs:2131:45
     |
2131 |             ["Size of ProfilingStackFrame"][::std::mem::size_of::<ProfilingStackFrame>() - 32usize];
     |                                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ attempt to compute `28_usize - 32_usize`, which would overflow
```

### OpenHarmony

```
make[4]: *** No rule to make target '/home/runner/.cargo/git/checkouts/mozjs-fa11ffc7d4f1cc2d/18ab014/mozjs-sys/mozjs/config/nsinstall.c', needed by 'host_nsinstall.o'.  Stop.
```

--- a/.github/workflows/android.yml
+++ b/.github/workflows/android.yml
@@ -28,6 +28,8 @@ env:
   # SCCACHE_GHA_ENABLED: "true"
   # RUSTC_WRAPPER: "sccache"
   # CCACHE: "sccache"
+  # Increment this to purge the cache (https://github.com/mozilla/sccache/blob/main/docs/GHA.md)
+  SCCACHE_GHA_VERSION: 2
   CARGO_INCREMENTAL: 0
 
 jobs:

--- a/.github/workflows/android.yml
+++ b/.github/workflows/android.yml
@@ -24,10 +24,9 @@ on:
 env:
   RUST_BACKTRACE: 1
   SHELL: /bin/bash
-  # FIXME: Temporarily commented to see if it helps with #34571.
-  # SCCACHE_GHA_ENABLED: "true"
-  # RUSTC_WRAPPER: "sccache"
-  # CCACHE: "sccache"
+  SCCACHE_GHA_ENABLED: "true"
+  RUSTC_WRAPPER: "sccache"
+  CCACHE: "sccache"
   # Increment this to purge the cache (https://github.com/mozilla/sccache/blob/main/docs/GHA.md)
   SCCACHE_GHA_VERSION: 2
   CARGO_INCREMENTAL: 0
@@ -58,9 +57,8 @@ jobs:
         with:
           ref: ${{ github.event.pull_request.head.sha }}
           fetch-depth: 2
-      # FIXME: Temporarily commented to see if it helps with #34571.
-      # - name: Run sccache-cache
-      #   uses: mozilla-actions/sccache-action@v0.0.6
+      - name: Run sccache-cache
+        uses: mozilla-actions/sccache-action@v0.0.7
       - name: Install crown
         run: cargo install --path support/crown
       - name: Bootstrap Python

--- a/.github/workflows/lint.yml
+++ b/.github/workflows/lint.yml
@@ -7,6 +7,8 @@ env:
   RUST_BACKTRACE: 1
   SHELL: /bin/bash
   SCCACHE_GHA_ENABLED: "true"
+  # Increment this to purge the cache (https://github.com/mozilla/sccache/blob/main/docs/GHA.md)
+  SCCACHE_GHA_VERSION: 2
   RUSTC_WRAPPER: "sccache"
   CCACHE: "sccache"
   CARGO_INCREMENTAL: 0

--- a/.github/workflows/linux.yml
+++ b/.github/workflows/linux.yml
@@ -75,6 +75,8 @@ on:
 env:
   RUST_BACKTRACE: 1
   SHELL: /bin/bash
+  # Increment this to purge the cache (https://github.com/mozilla/sccache/blob/main/docs/GHA.md)
+  SCCACHE_GHA_VERSION: 2
 
 jobs:
   # Runs the underlying job (“workload”) on a self-hosted runner if available,

--- a/.github/workflows/mac.yml
+++ b/.github/workflows/mac.yml
@@ -52,6 +52,8 @@ on:
 env:
   RUST_BACKTRACE: 1
   SHELL: /bin/bash
+  # Increment this to purge the cache (https://github.com/mozilla/sccache/blob/main/docs/GHA.md)
+  SCCACHE_GHA_VERSION: 2
   SCCACHE_GHA_ENABLED: "true"
   RUSTC_WRAPPER: "sccache"
   CCACHE: "sccache"

--- a/.github/workflows/ohos.yml
+++ b/.github/workflows/ohos.yml
@@ -25,6 +25,8 @@ on:
 env:
   RUST_BACKTRACE: 1
   SHELL: /bin/bash
+  # Increment this to purge the cache (https://github.com/mozilla/sccache/blob/main/docs/GHA.md)
+  SCCACHE_GHA_VERSION: 2
   SCCACHE_GHA_ENABLED: "true"
   RUSTC_WRAPPER: "sccache"
   CCACHE: "sccache"

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -87,7 +87,7 @@ markup5ever = "0.14"
 memmap2 = "0.9.5"
 mime = "0.3.13"
 mime_guess = "2.0.5"
-mozangle = "0.5.2"
+mozangle = "0.5.3"
 net_traits = { path = "components/shared/net" }
 nix = "0.29"
 num-traits = "0.2"

--- a/deny.toml
+++ b/deny.toml
@@ -150,6 +150,9 @@ skip = [
 
     # some non-servo crates still use 0.14
     "glow",
+
+    # wgpu crates still depend on 1.1.0
+    "rustc-hash",
 ]
 
 # github.com organizations to allow git sources for

